templates/documents/create.html.twig

```diff
@@ -2,52 +2,46 @@
 
-    <main role="main" class="col-12 mt-3">
+    {{ form_start(document_form) }}
+
+    <main role="main" class = "col-12 mt-3">
         <form action="">
             <div class="form-group">
-                <label for="">{% trans %}Document name{% endtrans %}</label>
-                <input type="text" class="form-control" name="" id="" aria-describedby="helpId" placeholder="">
-                <small id="helpId" class="form-text text-muted">Provide a recognizable name for the document</small>
+                <label >{% trans %}Document name{% endtrans %}</label>
+                {{ form_widget(document_form.documentName, {'attr': {'type': 'text','class': 'form-control', 'aria-describedby': 'helpNameId', 'required': true}}) }}
+                <small id="helpNameId" class="form-text text-muted">Provide a recognizable name for the document</small>
             </div>
+
             <div class="form-group">
-                <label for="">{% trans %}Select a document type{% endtrans %}</label>
-                <select class="form-control" name="" id="">
-                    <!-- TODO: What kind of types do we have here? Should they be qustomisable? -->
-                    <option>Type 1</option>
-                    <option>Type 2</option>
-                    <option>Type 3</option>
-                </select>
+                <label >{% trans %}Select a document type{% endtrans %}</label>
+                <!-- TODO: What kind of types do we have here? Should they be customisable? -->
+                {{ form_widget(document_form.type, {'attr': {'class': 'form-control', 'aria-describedby': 'helpTypeId', 'required': true}}) }}
+                <small id="helpTypeId" class="form-text text-muted">Provide a document type</small>
             </div>
+
             <div class="form-group">
-                <label for="">{% trans %}Chose a file{% endtrans %}</label>
-                <input type="file" class="form-control-file" name="" id="" placeholder="" aria-describedby="fileHelpId">
-                <!-- TODO: Make sure to write out correct size and fileformats -->
-                <small id="fileHelpId" class="form-text text-muted">{% trans %}Max file size: 10mb. Fileformats accepted: .pdf, .txt, .md, .jpg, .png, .webp{% endtrans %}</small>
+                <label >{% trans %}Chose a file{% endtrans %}</label>
+                <!-- TODO: What kind of types do we have here? Should they be customisable? -->
+                {{ form_widget(document_form.filename, {'attr': {'type': 'file', 'class': 'form-control-file', 'aria-describedby': 'helpFileId', 'required': true}}) }}
+                <small id="helpFileId" class="form-text text-muted">{% trans %}Max file size: 10mb. Fileformats accepted: .pdf, .txt, .mp4, .jpeg, .png, .doc, .xls{% endtrans %}</small>
             </div>
+
+            {{ form_widget(document_form.uploadDocument, {'attr': {'class': 'btn btn-primary', 'type': 'submit'}}) }}
+            {{ form_rest(document_form) }}
```

Comment:
I think this will produce an extra `</form>` element.

Reply:
Yes. Avoided duplicate `<form>` elements.